osrm-backend/Dockerfile

```diff
@@ -5,8 +5,9 @@ RUN wget -q -O - https://ppa.openerp-experts.net/ppa.pub | apt-key add -
 RUN echo "deb [arch=amd64] https://ppa.openerp-experts.net/ trusty main" > /etc/apt/sources.list.d/osrm.list
 RUN apt-get update && apt-get install -y osrm.backend
 
-VOLUME ["/opt/osrm/data"]
+VOLUME ["/var/lib/osrm/data"]
 COPY osrm.conf /opt/osrm/etc/
 
 EXPOSE 5000
-ENTRYPOINT ["usr/bin/osrm-routed", "-c", "/opt/osrm/etc/osrm.conf", "/opt/osrm/data/map.osrm"]
+ENTRYPOINT ["usr/bin/osrm-routed"]
+CMD ["-c", "/opt/osrm/etc/osrm.conf", "/var/lib/osrm/data/map.osrm"]
```

Comment:
I would keep entrypoint args in the entrypoint array, as it is more clear and belong to to it.